### DISCUSSION TITLE
Quale dispositivo AlexaMediaPlayer

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#542] [HUB: Home Assistant]
E' possibile in uno script stabilire che un messaggio di AlexaMediaPlayer venga _trasmesso_ dal dispositivo a cui si sta parlando? Come si fa?

**Comment by Larry-78** [comment]
Ciao, prova a dare una occhiatà qui [Last called device](https://github.com/custom-components/alexa_media_player/wiki#last-called-device)

**Reply by Dregi56** [reply]
Grazie.

**Reply by Larry-78** [reply]
Di nulla 👍🏻
